### DISCUSSION TITLE
DLTA-AI : Data Labeling, Tracking and Annotation with AI

### DISCUSSION THREAD

**Discussion by 0ssamaak0** [open-mmlab/mmdetection#10381] [Show and tell]
Based on the amazing library of mmdetection models, DLTA-AI provides a fully automated workflow for image and video annotation, with support for both segmentation and tracking, and zero shot segmetnation and tracking as well (with SAM)

 I hope to hear your feedback and criticism
 
[Repo Link](https://github.com/0ssamaak0/DLTA-AI)
[Video](https://www.youtube.com/watch?v=nHyxmDn6CzE)

![image](https://github.com/0ssamaak0/DLTA-AI/blob/master/assets/gif_main.gif?raw=true)

**Comment by hhaAndroid** [comment]
@0ssamaak0 Thank you very much. Can you tell me what is the difference between DLTA-AI and labelstudio? We have made a simple [demo](https://github.com/open-mmlab/playground/tree/main/label_anything), what should we do if we want to connect to DLTA-AI? Thank you!

**Reply by 0ssamaak0** [reply]
Regarding the differences:

I think the main difference is the customizability, user can choose from (+50) models for segmentation tasks, export formats can be user-defined too
Beside this:
- the video mode is very user friendly and convenient
- the integration of SAM to DLTA-AI provides many options, like: tracking with SAM, and enhancing the segmentations of other models with SAM

I really invite you to check the README for more details, and try it yourself

Regarding the second question: I didn't get it, what do you mean by: connecting to DLTA-AI?

Thank you 😃

**Reply by 0ssamaak0** [reply]
If you mean by connecting: using mmdetection API, it's already implemented, the **Model Explorer** provides almost all mmdetection v2 models to select from

**Reply by hhaAndroid** [reply]
> If you mean by connecting: using mmdetection API, it's already implemented, the **Model Explorer** provides almost all mmdetection v2 models to select from

OK.
